packages/react/src/LayoutExamples/KitchenSink.tsx

```diff
@@ -46,6 +48,23 @@ export function KitchenSink(props: KitchenSinkProps) {
           }}
           align="center"
         />
+
+        <div
+          style={{display: 'flex', justifyContent: 'center', margin: 'var(--brand-Hero-narrow-padding)', marginTop: 0}}
+        >
+          <Text>
+            To show or hide the grid, click within the frame, then press&nbsp;
+            <code
+              style={{
+                padding: '0.25rem 0.5rem',
+                backgroundColor: 'var(--brand-color-canvas-subtle)'
+              }}
+            >
+              ctrl + shift + L
+            </code>
+          </Text>
+        </div>
```

Comment:
I'm not sure this should go in here 🤔 .. as this functions a recipe layout for Primer components. Instead, could we turn the DevGrid into a global decorator? Down the line, I can also see this toggle-able through the Storybook toolbar via a Storybook addon.

Reply:
> Gutters: We have two different kind of gutters, normal and spacious, and it would be great to be able to account for both…! Also, what happens on mobile? Will we keep to 12 columns even on `sm` and `md`? Feels like it could be very stripy… but maybe it makes sense?

I guess it really depends on how the designers utilize the grid in mobile, right?